.gitignore

```diff
@@ -13,6 +13,3 @@
 node_modules
 checkstyle.xml
 loopback-connector-*.tgz
```

Comment:
@candy Actually it was confusing me too; it does have EOL... GH does not show it here.